<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>主窗口</string>
  </property>
  <property name="windowIcon">
   <iconset resource="SvpClient.qrc">
    <normaloff>:/icons/remote-desktop.png</normaloff>:/icons/remote-desktop.png</iconset>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextBesideIcon</enum>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>800</width>
         <height>566</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionFullScreen"/>
   <addaction name="actionSpecialKey"/>
   <addaction name="actionRefresh"/>
   <addaction name="actionStatistics"/>
   <addaction name="actionConfigure"/>
   <addaction name="actionAbout"/>
  </widget>
  <action name="actionFullScreen">
   <property name="icon">
    <iconset resource="SvpClient.qrc">
     <normaloff>:/icons/view-fullscreen.png</normaloff>:/icons/view-fullscreen.png</iconset>
   </property>
   <property name="text">
    <string>全屏</string>
   </property>
  </action>
  <action name="actionSpecialKey">
   <property name="icon">
    <iconset resource="SvpClient.qrc">
     <normaloff>:/icons/go-jump-locationbar.png</normaloff>:/icons/go-jump-locationbar.png</iconset>
   </property>
   <property name="text">
    <string>特殊按键</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="SvpClient.qrc">
     <normaloff>:/icons/view-refresh.png</normaloff>:/icons/view-refresh.png</iconset>
   </property>
   <property name="text">
    <string>刷新</string>
   </property>
  </action>
  <action name="actionStatistics">
   <property name="icon">
    <iconset resource="SvpClient.qrc">
     <normaloff>:/icons/view-statistics.png</normaloff>:/icons/view-statistics.png</iconset>
   </property>
   <property name="text">
    <string>统计信息</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="icon">
    <iconset resource="SvpClient.qrc">
     <normaloff>:/icons/configure.png</normaloff>:/icons/configure.png</iconset>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="SvpClient.qrc">
     <normaloff>:/icons/help-about.png</normaloff>:/icons/help-about.png</iconset>
   </property>
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="actionRemote">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>适应虚拟机桌面</string>
   </property>
  </action>
  <action name="actionCenter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>在滚动区域居中</string>
   </property>
  </action>
  <action name="actionClient">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>适应客户端窗口</string>
   </property>
  </action>
  <action name="actionScale">
   <property name="icon">
    <iconset resource="SvpClient.qrc">
     <normaloff>:/icons/transform-scale.png</normaloff>:/icons/transform-scale.png</iconset>
   </property>
   <property name="text">
    <string>缩放</string>
   </property>
  </action>
  <action name="actionUSB">
   <property name="icon">
    <iconset resource="SvpClient.qrc">
     <normaloff>:/icons/drive-usb.png</normaloff>:/icons/drive-usb.png</iconset>
   </property>
   <property name="text">
    <string>USB</string>
   </property>
   <property name="toolTip">
    <string>USB</string>
   </property>
  </action>
  <action name="actionEmpty">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>  (空)  </string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="SvpClient.qrc"/>
 </resources>
 <connections/>
</ui>
